autoload/vundle/scripts.vim

```diff
@@ -254,4 +254,33 @@ func! s:load_scripts(bang)
   return eval(readfile(f, 'b')[0])
 endf
 
+
+" ---------------------------------------------------------------------------
+" Get the dependencies of the given bundle.
+"
+" bundle -- The bundle to check. (must be installed)
+" return -- List of bundle names.
+" ---------------------------------------------------------------------------
+func! vundle#scripts#getdeps(bundle)
+  if !empty(glob(a:bundle['rtpath'] . '/addon-info.json'))
```

Comment:
Why glob? It is rather unlikely that we will have wildcards in the rtpath, so how about:

```viml
if filereadable(a:bundle['rtpath'] . '/addon-info.json')
```